Formula/p/pikchr.rb

```diff
@@ -0,0 +1,22 @@
+class Pikchr < Formula
+  desc "PIC-like markup language for diagrams in technical documentation"
+  homepage "https://pikchr.org/"
+  url "https://pikchr.org/home/zip/ae3317b0ec/pikchr-ae3317b0ec.zip"
+  version "2024-02-12"
+  sha256 "b07c75d30756f3f19308da2261de85c847b5ea93d001db1341b9829376f7de75"
+  license "0BSD"
+
+  def install
+    system ENV.cc, "-DPIKCHR_SHELL", "-o", "pikchr", "pikchr.c", "-lm"
+    cp "pikchr", prefix
+    bin.mkpath
+    bin.install "pikchr"
```

Comment:
maybe have upstream add a makefile?

Reply:
They do have a Makefile, though the way I do it here is recommended by them.  I could change it to use the Makefile if you think it's better; it would work as well but it would be a bit slower

Reply:
any reference for the installation recommendation? 

a bit slower is fine, at least the installation step is easy to read.

Reply:
I did change the installation step to use make